addons/event/static/src/js/tours/event_tour.js

```diff
@@ -38,49 +38,54 @@ tour.register('event_tour', {
     edition: 'community',
 }, {
     trigger: '.o-kanban-button-new',
+    extra_trigger: '.o_event_kanban_view',
     content: _t("Let's create your first <b>event</b>."),
     position: 'bottom',
     width: 175,
 }, {
     trigger: 'input[name="name"]',
+    extra_trigger: '.o_event_form_view',
     content: _t("This is the <b>name</b> your guests will see when registering."),
     run: 'text Odoo Experience 2020',
 }, {
     trigger: 'input[name="date_end"]',
```

Comment:
Same remark, could be in the trigger instead IMO, please also check other inputs.
It's always better to avoid using the extra_trigger when we can.